dashboard/dashboard.py

```diff
@@ -55,14 +138,23 @@ def subscribe_page() -> None:
             st.error("Invalid email address, try again!")
         else:
             st.success(f"Thank you for subscribing! Email: {email}")
-            st.write("Daily Reports Subscription:",
-                     "Yes" if subscribe_daily else "No")
+            st.write("Daily Reports Subscription: Yes")
             st.write("Alerts Subscription:",
                      "Yes" if subscribe_alerts else "No")
+            if selected_genres:
+                st.write("Subscribed Genres:", ", ".join(selected_genres))
+            else:
+                st.write("Subscribed Genres: None")
 
             print(f"Email: {email}")
-            print(f"Daily Reports: {subscribe_daily}")
             print(f"Alerts: {subscribe_alerts}")
+            print(f"Subscribed genres: {selected_genres}")
+
+            selected_genre_ids = convert_subscribed_genres_to_ids(
+                selected_genres, genres_dict)
+
+            add_subscriber_info_to_db(
+                email, subscribe_alerts, selected_genre_ids)
 
```

Comment:
As discussed, we could also add an option for users to unsubscribe

Reply:
implemented.